cookiecutter_project_upgrader/cli.py

```diff
@@ -11,6 +13,9 @@
 @click.option('--context-file', '-c', type=click.Path(file_okay=True, readable=True, allow_dash=True),
               default="docs/cookiecutter_input.json", help="Default: docs/cookiecutter_input.json")
 @click.option('--branch', '-b', default="cookiecutter-template", help="Default: cookiecutter-template")
+@click.option('--upgrade-branch', '-u',
+              help="Cookiecutter template git branch name to upgrade from?")
```

Comment:
```suggestion
              help="Optional branch name of cookiecutter template to checkout before upgrading.")
```
to avoid questions in helptexts